### ISSUE TITLE
History is not updated after any request

### ISSUE THREAD

**Issue by povazhny** [jarrodek/ChromeRestClient#1071]
The History is not updated after request executed. Need to restart application to update history.

## Expected outcome
Last request is at the top of the list.

## Actual outcome
Last request is NOT in the list.

## Versions
App: 10.0.3
Platform: 5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/61.0.3163.100 Safari/537.36

## Steps to reproduce
1. Perform any request.
2. Open History
3. See recent request list.

Workaround:
Restart the application. Recent requests appear on the list in correct order.

**Comment by jarrodek** [comment]
Hi,

Thank you for the issue report.
I can't reproduce the issue.
This is state before making a request:
![image](https://user-images.githubusercontent.com/3320020/31619690-2202ed08-b24a-11e7-98a3-ace301ddcc46.png)

An this is after:
![image](https://user-images.githubusercontent.com/3320020/31619740-40dbff8a-b24a-11e7-90ce-e88d2226ff61.png)

Can you see any errors in the console?

**Comment by povazhny** [comment]
Hi jarrodek,

Steps to reproduce

1. Send several different request, i.e. request #1, request #2, request #3.
2. Open History. Requests are logged in the following order: #3, #2, #1.
3. In the History select request #1 and re-send it again.

Expected:
Requests are logged in the following order: #1, #3, #2, #1.

Actual:
Requests are logged in the following order: #3, #2, #1.

If you restart the application - you will see #1, #3, #2, #1.

**Comment by povazhny** [comment]
If you send completely new request i.e. #4 - it will appear on the top in the History list immediately.

**Comment by povazhny** [comment]
But #4 will not appear on the right History pane until application restart.

**Comment by jarrodek** [comment]
I've just discovered that the history panel is missing implementation of one event send by the data store so it might not catch all changes. It wasn't important before because each time you opened the history panel new query was performed to get newest data.
In current version behavior is different. Data are not refreshed each time you open the panel and therefore missing event handler became a problem. 
I've fixed that just a minute ago and now it's being tested on Travis. I'll update the app today.
